libvmi/driver/xen/xen_events.c

```diff
@@ -728,6 +728,13 @@ status_t xen_init_events_new(vmi_instance_t vmi)
     vmi->driver.set_cpuid_event_ptr = &xen_set_cpuid_event;
     vmi->driver.set_debug_event_ptr = &xen_set_debug_event;
 
+    vmi->driver.slat_get_domain_state_ptr = &xen_altp2m_get_domain_state;
```

Comment:
Wiring up these functions should be done in xen.c and only if the Xen version is 4.6 or newer.